scripts/shared/post_mortem.sh

```diff
@@ -26,6 +26,14 @@ function post_analyze() {
 
     echo "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
 
+    namespace="submariner-operator"
+    for pod in $(kubectl get pods --selector=name=submariner-operator -n $namespace -o jsonpath='{.items[*].metadata.name}'); do
```

Comment:
The PR in itself is good. But I have a feeling that we will be dumping too many logs during post-mortem. Let's keep an eye on the logs and if we feel something is logged frequently, we can make the necessary changes in the code.

Reply:
Yeah, we could remove logs later. But I find so many times that I want something from post mortem and it isn't there ':-)